packages/editor/example/index.html

```diff
@@ -17,6 +17,11 @@
   <div class="nav-wrapper" id="toolbar">
     <a href="#" class="brand-logo"><strong>User and Task</strong> Editor</a>
     <div id="nav-mobile" class="right hide-on-med-and-down">
+      <button
+              id="model-data-button"
+              class="btn">
+        Model Data
```

Comment:
Is thie the upload button for the model schema? I'd relabel it then to 'Upload Model/Domain Schema' or something similar.